data/helpers.d/postgresql

```diff
@@ -282,11 +282,11 @@ ynh_psql_test_if_first_run() {
 
 		sudo --login --user=postgres psql -c"ALTER user postgres WITH PASSWORD '$psql_root_password'" postgres
 
-		# force all user to connect to local database using passwords
+		# force all user to connect to local databases using hashed passwords
 		# https://www.postgresql.org/docs/current/static/auth-pg-hba-conf.html#EXAMPLE-PG-HBA.CONF
 		# Note: we can't use peer since YunoHost create users with nologin
 		#  See: https://github.com/YunoHost/yunohost/blob/unstable/data/helpers.d/user
-		ynh_replace_string --match_string="local\(\s*\)all\(\s*\)all\(\s*\)peer" --replace_string="local\1all\2all\3password" --target_file="$pg_hba"
+		ynh_replace_string --match_string="local\(\s*\)all\(\s*\)all\(\s*\)peer" --replace_string="local\1all\2all\3md5" --target_file="$pg_hba"
```

Comment:
Another question though : are we confident that applications can use the hash-auth mechanism transparently ? I suppose any reasonnable app uses a lib which I guess is supposed to be able to handle that ?

Reply:
Yes. md5 is a very very old authentication technique in PostgreSQL and will be handle by all tools.
Things can be different for scram-sha-256, but this is more a question about how we generate the password than a line in the pg_hba file, since `md5` method accepts scram-sha-256.